index.js

```diff
@@ -29,7 +29,13 @@ const buildCurrent = helpers.getBuildNumberFromPlist(pathToPlist);
 const build = buildCurrent + 1;
 
 // getting commit message
-const message = argv.m || argv.message || `release ${version}: increase versions and build numbers`;
+let message
+if(argv.m || argv.message) {
+  message = argv.m || argv.message
+  message = message.replace('${version}', version)
+} else {
+  message = `release ${version}: increase versions and build numbers`;
+}
```

Comment:
Maybe it will be better to use .replace for both cases?
```javascript
const messageTemplate = argv.m || argv.message
  || 'release ${version}: increase versions and build numbers';
const message = messageTemplate.replace('${version}', version);
```

Reply:
sure that probably makes it clearer, would just be nice to have it available in custom commits, just go ahead and use whatever implementation you like.